### PR TITLE
Add a function to pass view=preview to draft entry statuses

### LINKED ISSUE #344: Pass "view=preview" to Entry view site when Entry is in draft

### PR DESCRIPTION
**Pull request by ArsalanSavand**
Fixes #344

### CODE REVIEW

File: app/entry/entry_edit/entry_edit_controller.js [hunk 1]
```diff
@@ -135,6 +135,16 @@ function EntryEditController($scope, $rootScope, $state, $stateParams, $mdToast,
     );
   };
 
+  $scope.getEntryUrl = function () {
```
Comment (amirm3hdi) [inline app/entry/entry_edit/entry_edit_controller.js]: Requires documentation.

File: app/entry/entry_edit/entry_edit_controller.js [hunk 2]
```diff
@@ -135,6 +135,16 @@ function EntryEditController($scope, $rootScope, $state, $stateParams, $mdToast,
...
+    var params = "";
+
+    if ($scope.form.status === 0) {
```
Comment (amirm3hdi) [inline app/entry/entry_edit/entry_edit_controller.js]: Use `Codekit` service for the status id.

File: app/entry/entry_edit/entry_edit_controller.js (hunk 1 above)
Comment (ArsalanSavand) [inline app/entry/entry_edit/entry_edit_controller.js]: Done.

File: app/entry/entry_edit/entry_edit_controller.js (hunk 2 above)
Comment (ArsalanSavand) [inline app/entry/entry_edit/entry_edit_controller.js]: Done.

File: app/entry/entry_edit/entry_edit_controller.js [hunk 3]
```diff
@@ -136,6 +136,20 @@ function EntryEditController($scope, $rootScope, $state, $stateParams, $mdToast,
   };
 
   /**
+   * @method getEntryUrl
+   * @desc add draft parameters if entry is draft
```
Comment (amirm3hdi) [inline app/entry/entry_edit/entry_edit_controller.js]: Add not add.

Comment (ArsalanSavand) [inline app/entry/entry_edit/entry_edit_controller.js]: Done.

**Pull request merged**